JoinColony/colonyCDapp
# Cancelling staked expenditure

## Description

A staked expenditure can be cancelled by anyone with arbitration permissions or using motions (#1061) and a choice can be made whether to punish the creator by forfeiting their stake.

## Testing
1. Enable reputation monitor. Pay yourself some tokens and activate them.
2. Install and enable Staked Expenditure extension.
3. Create an expenditure by staking it and lock it.
4. Cancel staked expenditure button should appear. In the dialog that it opens you should be able to select whether to punish the creator.
5. `cancelStakedExpenditure` saga should fire. After page refresh, the expenditure status should change to Cancelled. 
6. If you chose not to punish the creator, you should be able to reclaim the stake.

Resolves #1033
